core/src/templates/components/alert/alert.twig

```diff
@@ -19,13 +19,13 @@
  * - alert_body: Text details of the alert message.
  */
 #}
-<div {{ attributes }} class="su-alert {{ modifier_class }}" >
-  <div class="su-alert__body">
```

Comment:
I'm curious as to why there's two levels of <div> would one be sufficient?

Reply:
Good question. There may have been a reason for this but it doesn't come to mind. I would have to investigate further. Can you provide a recommendation? Assume it was an oversight/overbuild.

Reply:
On further inspection, it looks like the nested `<div>s` are to help with layout.